package/MDAnalysis/analysis/waterdynamics.py

```diff
@@ -766,14 +767,18 @@ def _getOneDeltaPoint(self,universe, repInd, i ,t0, dt):
             valHH += self.lg2(np.dot(unitHHVector0,unitHHVectorp))
             valdip +=  self.lg2(np.dot(unitdipVector0,unitdipVectorp))
             n += 1
-        valOH = valOH/n
-        valHH = valHH/n
-        valdip = valdip/n
-        return (valOH,valHH,valdip)
+
+        if n == 0:
+            return (0,0,0)
```

Comment:
Try to keep a limit at 80-ish columns. There is no point at cutting lines at 79 for the sake of it, you 82 or 85 as just as readable, but long lines are often the sign of other issues.